src/main/kotlin/no/nav/helse/prosessering/v1/MeldingV1.kt

```diff
@@ -8,58 +9,55 @@ import java.time.LocalDate
-    val harBekreftetOpplysninger : Boolean,
+    @JsonAlias("bekrefter_periode_over_8_uker") val bekrefterPeriodeOver8Uker: Boolean? = null,
+    @JsonAlias("utenlandsopphold_i_perioden") val utenlandsoppholdIPerioden: UtenlandsoppholdIPerioden?,
+    @JsonAlias("ferieuttak_i_perioden") val ferieuttakIPerioden: FerieuttakIPerioden?,
+    @JsonAlias("har_medsoker") val harMedsøker : Boolean,
+    @JsonAlias("samtidig_hjemme") val samtidigHjemme: Boolean? = null,
+    @JsonAlias("har_forstatt_rettigheter_og_plikter") val harForståttRettigheterOgPlikter : Boolean,
+    @JsonAlias("har_bekreftet_opplysninger") val harBekreftetOpplysninger : Boolean,
     val tilsynsordning: Tilsynsordning?,
     val beredskap: Beredskap?,
-    val nattevaak: Nattevaak?,
+    @JsonAlias("nattevaak") val nattevåk: Nattevaak?,
     val frilans: Frilans?,
-    val selvstendigVirksomheter: List<Virksomhet>? = null,
-    @JsonProperty("skal_bekrefte_omsorg") val skalBekrefteOmsorg: Boolean? = null, // TODO: Fjern optional når prodsatt.
-    @JsonProperty("skal_passe_pa_barnet_i_hele_perioden") val skalPassePaBarnetIHelePerioden: Boolean? = null, // TODO: Fjern optional når prodsatt.
-    @JsonProperty("beskrivelse_omsorgsrollen") val beskrivelseOmsorgsRollen: String? = null // TODO: Fjern optional når prodsatt.
+    @JsonAlias("selvstendig_virksomheter") val selvstendigVirksomheter: List<Virksomhet>? = null,
+    @JsonAlias("skal_bekrefte_omsorg") val skalBekrefteOmsorg: Boolean? = null, // TODO: Fjern optional når prodsatt.
+    @JsonAlias("skal_passe_pa_barnet_i_hele_perioden") val skalPassePaBarnetIHelePerioden: Boolean? = null, // TODO: Fjern optional når prodsatt.
+    @JsonAlias("beskrivelse_omsorgsrollen") val beskrivelseOmsorgsrollen: String? = null // TODO: Fjern optional når prodsatt.
 )
 
 data class Virksomhet(
     val næringstyper: List<Næringstyper>,
-    @JsonProperty("fiskerErPåBladB")
+    @JsonAlias("fiskerErPåBladB")
     val fiskerErPåBladB: Boolean? = null,
     @JsonFormat(pattern = "yyyy-MM-dd")
-    @JsonProperty("fraOgMed")
+    @JsonAlias("fraOgMed")
     val fraOgMed: LocalDate,
     @JsonFormat(pattern = "yyyy-MM-dd")
-    @JsonProperty("tilOgMed")
+    @JsonAlias("tilOgMed")
     val tilOgMed: LocalDate? = null,
     val næringsinntekt: Int? = null,
-    @JsonProperty("navnPåVirksomheten")
+    @JsonAlias("navnPåVirksomheten")
     val navnPåVirksomheten: String,
     val organisasjonsnummer: String? = null,
-    @JsonProperty("registrertINorge")
+    @JsonAlias("registrertINorge")
     val registrertINorge: Boolean,
-    @JsonProperty("registrertIUtlandet")
+    @JsonAlias("registrertIUtlandet")
     val registrertIUtlandet: Land? = null,
-    @JsonProperty("yrkesaktivSisteTreFerdigliknedeÅrene")
+    @JsonAlias("yrkesaktivSisteTreFerdigliknedeÅrene")
     val yrkesaktivSisteTreFerdigliknedeÅrene: YrkesaktivSisteTreFerdigliknedeÅrene? = null,
-    @JsonProperty("varigEndring")
+    @JsonAlias("varigEndring")
```

Comment:
Samme som forrige.
Burde ikke alias isåfall være snake_case?

Reply:
Fjernet

Reply:
👍